shapely/geos.py

```diff
@@ -62,6 +62,8 @@ def load_dll(libname, fallbacks=None, mode=DEFAULT_MODE):
                 libname, fallbacks or []))
 
 _lgeos = None
+is_conda = os.path.exists(os.path.join(sys.prefix, 'conda-meta'))
```

Comment:
What would you think about defining a function instead @arubiales ? Use of a function will make this easier to test and we'll be able to avoid checking the filesystem in some cases.

```suggestion
def exists_conda_env():
    """Does this module exist in a conda environment?"""
    return os.path.exists(os.path.join(sys.prefix, 'conda-meta'))
```

Reply:
Ohhh, I have acepted but we need to implement the function in the three checks! I'm going to change it!

Thanks.